app/test/integration/launch-test.ts

```diff
@@ -28,36 +21,38 @@ describe('App', function(this: any) {
     if (process.platform === 'win32') {
       appPath += '.cmd'
     }
+
+    const root = path.resolve(__dirname, '..', '..', '..')
+
     app = new Application({
       path: appPath,
-      args: [path.join(__dirname, '..', '..', '..', 'out')],
+      args: [path.join(root, 'out')],
     })
     return app.start()
   })
 
-  beforeEach(function() {
-    chaiAsPromised.transferPromiseness = app.transferPromiseness
-  })
-
   afterEach(function() {
     if (app && app.isRunning()) {
       return app.stop()
     }
+    return Promise.resolve()
   })
 
-  it('opens a window on launch', function() {
-    return app.client
-      .waitUntil(() => app.browserWindow.isVisible(), 5000)
-      .getWindowCount()
-      .should.eventually.equal(1)
-      .browserWindow.isMinimized()
-      .should.eventually.be.false.browserWindow.isDevToolsOpened()
```

Comment:
`isDevToolsOpened()` isn't exposed in the `spectron` typings because they just wrap `electron` - more context here: https://github.com/electron/spectron/issues/343